packages/docs/pages/reference/elements/h.mdx

```diff
@@ -8,12 +8,51 @@ import {
 
 # `<h>`
 
-Short description of what the `<h>` tag does.
+The `<h>` tag is an immediate child of the index tag, [`<idx>`](./idx), used to
+create the name or symbol to appear on the Index page.
 
 ## Syntax
 
 <SyntaxDisplay name="h" />
 
 ## Examples
 
-    
\ No newline at end of file
+Assuming the presence of the terms `group`, `subgroup`, `normal subgroup` within
+the text, the following `<h>` tags would be used to create standalone entries in
+the Index for the terms `group` and`subgroup`, and an entry for`normal`
+appearing as a secondary entry under `subgroup` in the Index.
+
+```ptx-example
+<FRAGMENT parents="p">
+<idx><h>group</h></idx>
+<idx><h>subgroup</h></idx>
+<idx><h>subgroup</h><h>normal</h></idx>
+</FRAGMENT>
+```
+
+Assuming the notation <m>\trianglelefteq</m> has been introduced to indicate a
+normal subgroup, the following `<h>` tag would create an entry for that symbol
+under the heading `notation` and symbol would appear in the list under
+`notation` sorted by the keyword normal, and with the cross-reference to see
+`normal subgroup` for further information.
+
+```ptx-example
+<FRAGMENT parents="p">
+<idx><h>notation</h><h sortby="normal"><m>\trianglelefteq</m></h>
+<see>normal subgroup</see></idx>
+</FRAGMENT>
+```
+
+In the example above, we gave a reference to a `normal subgroup`, but the only
+Index entry is to `normal` as a subcategory to `subgroup`.  To clarify this, we
+may want to include either a standalone entry for `normal subgroup`, or perhaps if one is also going to discuss normal field extensions, to
+include it as a secondary entry under `normal`.
+
+```ptx-example
+<FRAGMENT parents="p">
+<idx><h>normal subgroup</h></idx>
+
+<idx><h><normal</h><h>field extension</h><seealso>Galois</seealso></idx>
+<idx><h><normal</h><h>subgroup</h></idx>
```

Comment:
If I understand correctly, these examples won't render anything because they only show up in the index? If so, I think rendering the index should be part of the example.